Refactor

lainsworth8801 opened the pull request:
Refactored openstudio-web to a static page:
1. removed search bar on main site;
2. removed developer email list;
3. added latest os downloads and news releases.

lainsworth8801 committed:
Initial check in

axelstudios committed:
Added OpenStudio Contribution Policy

lainsworth8801 committed:
Removed developer email list and added latest downloads and news releases.

lainsworth8801 committed:
Spreaded out navbar tabs

kflemin reviewed (approved):
looks great! 
1. Can you change the colors of the links and buttons to match the old site?
2. Have the "Contact Us" footer link point to the forum page.  Remove the 'contact' page.
Thanks!

lainsworth8801 committed:
Point "contact us" to unmet forum; button background color.

lainsworth8801 committed:
Clean up typo, add logos.

axelstudios committed:
UI fixes
- Reflow fixes
- Lint fixes
- Simplified publication template
- Fixed publication overflow
- Removed logging
- Fixed url redirects
- Localized social media images
- Removed Google+
- Removed style attributes

axelstudios committed:
Localized fonts
